records/scrum_team_charter.md

```diff
@@ -0,0 +1,87 @@
+# Visual Meaning Scrum Team Charter
+
+## Things we always do
+
+- Try to write the best code
+
+- **Be kind**
+  - Assume best intent
+  - Bring humanity to work - full spectrum of emotion and acknowledgement that we are humans who do have personal lives
+  - Reach out and be vulnerable
+
+- **Commit**
+  - Do what we say we will do (and communicate when things change)
+  - Keep practical details front of mind
+  - Prioritise team outcomes
+  - Be meaningfully wrong
+
+- **Collaborate**
+  - Teach, share information and decisions, learning early and often and recording this where people can find the information again
+  - Broadcast by default
+
+- **Engage**
+  - Engage in synchronous sessions
+  - Thumbs up/down in response to questions in a group
+  - Aim to ask questions first (curiosity, clean/clean-ish questions)
+  - Never hesitate to express ourselves
+
+## Things we never do
+
+- **During delivery**
+  - Land code without review
+  - Hide information or code
+  - Criticise the person instead of the code
+  - Go it alone when doing something outside of your comfort zone
+
+- **During communication**
+  - Make assumptions about others' behaviour (other than best intent)
+  - Bottle up frustrations without discussing them
+  - Raise voices to each other
+  - Be vague (share in a way others can't engage or disagree with)

```

Comment:
this is subjective. I have no idea how people with understand me. I will try to reduce sharing in general. Yes I also received feedback that because I list to many issues the other person become disengaged.